indico/modules/rb_new/client/js/common/filters/validation.js

```diff
@@ -0,0 +1,45 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {toMoment} from 'indico/utils/date';
+import {Translate} from 'indico/react/i18n';
+import {handleAxiosError} from 'indico/utils/axios';
+import {resetPageState} from '../../actions';
+
+
+const mixedValidator = (filters) => {
+    let valid = true;
+    if (!('timeSlot' in filters)) {
+        return valid;
+    }
+
+    const {timeSlot: {startTime: st, endTime: et}} = filters;
+    if (st && et && toMoment(st, 'HH:mm').isSameOrAfter(toMoment(et, 'HH:mm'))) {
```

Comment:
`HH:mm` can be rreplaced with `moment.HTML5_FMT.TIME`

Reply:
What about making `HH:mm` the  default value for `format` in `toMoment`?